app/components/forms/orders/order-form.js

```diff
@@ -29,8 +29,8 @@ export default Component.extend(FormMixin, {
   buyerHasFirstName : readOnly('data.user.firstName'),
   buyerHasLastName  : readOnly('data.user.lastName'),
   holders           : computed('data.attendees', 'buyer', function() {
-    this.data.attendees.forEach((attendee, index) => {
-      if (index === 0 && this.buyerFirstName && this.buyerLastName) {
+    this.data.attendees.forEach(attendee => {
+      if (this.buyerFirstName && this.buyerLastName) {
```

Comment:
Not sure whether we should deal with this here, but we might have people without first/last name distinction. They just have one name.